docs/docs/customization.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Customizing ZMK/`zmk-config` folders
```

Comment:
Needs to be rebased after #1280?

Reply:
Rebased. Not sure why that would have been showing up as a change in this PR.